### PR TITLE
NCBC-2911 update travel-sample docs and attributes

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

File: modules/concept-docs/partials/attributes.adoc [hunk 1]
```diff
@@ -1,4 +1 @@
-:dotnet-api-link: http://docs.couchbase.com/sdk-api/couchbase-dotnet-client-3.0.0
-:dotnet-current-version: 3.0.0
-:version-server: 7.0
-
+include::project-docs:partial$attributes.adoc[]
```
Comment (RichardSmedley) [inline modules/concept-docs/partials/attributes.adoc:1]: Generally, we're deleting the unneeded attributes.adocs

**Review by RichardSmedley — CHANGES_REQUESTED** [review]
Afaict there are only two pages using the attributes.adoc
modules/hello-world/pages/webui-cli-access.adoc:include::partial$attributes.adoc[]
modules/howtos/pages/concurrent-document-mutations.adoc:include::partial$attributes.adoc[]
if we update them,
then all of the attributes adocs outside project-docs can be deleted.

**Comment by osfameron** [comment]
> Afaict there are only two pages using the attributes.adoc
> modules/hello-world/pages/webui-cli-access.adoc:include::partial$attributes.adoc[]
> modules/howtos/pages/concurrent-document-mutations.adoc:include::partial$attributes.adoc[]
> if we update them,
> then all of the attributes adocs outside project-docs can be deleted.

Nice spot - I'll ticket removing all the additional attributes.adoc files for this sprint after Tuesday for all SDKs.

**Comment by osfameron** [comment]
@RichardSmedley https://issues.couchbase.com/browse/DOC-8829

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Pull request merged**